docs/CHANGELOG.md

```diff
@@ -2,12 +2,37 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [13.0.0-beta.0](https://github.com/aeternity/aepp-sdk-js/compare/v12.1.3...v13.0.0-beta.0) (2023-03-01)
+## [13.0.0](https://github.com/aeternity/aepp-sdk-js/compare/v13.0.0-beta.0...v13.0.0) (2023-04-06)
 
 
 ### ⚠ BREAKING CHANGES
 
-Please check out [migration guide](./guides/migration/13.0.0.md).
+Please check out [migration guide](./guides/migration/13.0.0.md), and release notes for the [beta release](#1300-beta0-2023-03-01).
```

Comment:
Here are two approaches, to copy-paste the changelog from the last beta release adding current changes, or to refer the beta release. I'd prefer the second one to avoid duplication.

Reply:
both are fine I guess